db/migrations/mysql/000032_thread_memberships.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS ThreadMemberships;
```

Comment:
There is an inconsistency with the other down.sql. In the earlier one, we remove the indexes, columns and then the table. But here we are removing only the table. We should be consistent every where.

Reply:
Oops 👍